static/_redirects

```diff
@@ -87,6 +87,7 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 /docs/guides/securing-tcp /docs/capabilities/tcp
 /docs/guides/tcp.html /docs/guides/securing-tcp
 /guides/vs-code-server.html /docs/guides/code-server
+/docs/guides/vs-code-server.html /docs/guides/code-server
```

Comment:
This seems unrelated? I think this was from another PR that was merged already, so it should disappear if you sync the branch.

Reply:
Ah definitely a mistake. Thanks for pointing that one out.